src/components/home/modals/AliasAddress.vue

```diff
@@ -0,0 +1,319 @@
+            size="lg"
+            @click="addAlias">Add
+          </b-button>
+        </b-col>
+      </b-row>
+    </div>
+  </b-modal>
+</template>
+
+<script>
+import Vue from 'vue'
+import { MAX_ALIAS_LENGTH } from '@/constants'
+export default {
+    name: 'AliasAddress',
+    props: {
+        addresses: {
+            type: Object,
+            require: true,
+            default: function() {}
+        },
+        coldAddresses: {
+            type: Object,
+            require: true,
+            default: function() {}
+        },
+        address: {
+            type: String,
+            require: true,
+            default: ''
+        }
+    },
+    data() {
+        return {
+            curAlias: '',
+            curAddress: '',
+            aliasNum: 3,
+            aliasAddresses: {}
+        }
+    },
+    created() {
+        if (JSON.parse(window.localStorage.getItem(this.defaultAddress)) != null) {
+            this.aliasAddresses = JSON.parse(window.localStorage.getItem(this.defaultAddress)).alias
+        }
+    },
+    computed: {
+        defaultAddress() {
+            return Vue.ls.get('address')
+        },
+        isValidAddress() {
+            if (!this.curAddress) {
+                return void 0
+            }
+            return this.isExistedAddress
+        },
+        isExistedAddress() {
+            return this.curAddress in this.addresses || this.curAddress in this.coldAddresses
+        },
+        isExistedAlias() {
+            if (JSON.parse(window.localStorage.getItem(this.defaultAddress)) != null) {
+                let alias = JSON.parse(window.localStorage.getItem(this.defaultAddress)).alias
+                if (alias) {
+                    for (let key in alias) {
+                        if (this.curAlias.toLowerCase() === alias[key].toLowerCase()) {
+                            return true
+                        }
+                    }
+                }
+            }
+            return false
+        },
+        isAliasLessEight() {
+            return this.curAlias.length < MAX_ALIAS_LENGTH
+        },
+        isValidAlias() {
+            if (!this.curAlias) {
+                return void 0
+            }
+            return this.isAliasLessEight && !this.isExistedAlias
+        },
+        from3rdParty() {
+            return false
+        }
+    },
+    methods: {
+        resetData() {
+            this.aliasAddresses = JSON.parse(window.localStorage.getItem(this.defaultAddress)).alias
+            this.curAddress = ''
+            this.curAlias = ''
+        },
+        deleteAlias(addr) {
+            let userInfo = JSON.parse(window.localStorage.getItem(this.defaultAddress))
+            delete userInfo.alias[addr]
+            window.localStorage.setItem(this.defaultAddress, JSON.stringify(userInfo))
+            this.aliasAddresses = userInfo.alias
+        },
+        closeModal() {
+            this.resetData()
+            this.$refs.aliasAddressModal.hide()
```

Comment:
When the modal close, the addresses in left panel of home page should be updated. Currently not update unless we refresh the wallet.